SPECS/libdnf/libdnf.spec

```diff
@@ -92,6 +91,11 @@ Python 3 bindings for the hawkey library.
 # Allows cmake to find libsolv.
 find %{_prefix} -name "FindLibSolv.cmake" -exec cp {} cmake/modules ';'
 
+CFLAGS="`echo " %{build_cflags} " | sed 's/-Werror=format-security/-Wno-error=format-security/'`"
```

Comment:
Was this change intended?  do we need a comment to explain?

Reply:
Warnings are treated as errors. I will look into it and get a patch for next version.